app/helpers/work_packages_helper.rb

```diff
@@ -327,22 +327,33 @@ def work_package_show_attributes(work_package)
     group_work_package_attributes work_package_show_attribute_list(work_package)
   end
 
-  def work_package_show_table_row(attribute, klass = nil, attribute_lang = nil, value_lang = nil, &block)
```

Comment:
Sorry @myabc, but you are breaking plugins here by renaming the method (at least costs, probably more).

If I see this correctly the method-signature stays compatible, in that case we could use... how is it called, `deprecate_alias` or something?
Just make the old method available and call it deprecated ^^

edit: Moved the comment so that it is rendered at the bottom on GitHub... @houndci's comment on the other line made it vanish high up ^^